spec/support/schema_spec.rb

```diff
@@ -0,0 +1,35 @@
+WithSchema = Class.new do
+  include Surrealist
+
+  json_schema { { name: String, age: Integer } }
+
+  def name
+    'John'
+  end
+
+  def age
+    21
+  end
+end
+
+WithoutSchema = Class.new do
+  include Surrealist
+end
+
+RSpec.describe Surrealist do
+  describe '.defined_schema' do
+    context 'json schema defined' do
+      let(:instance) { WithSchema }
+      it do
```

Comment:
A space between `let` and `it` is missing.
Also I would suggest to add a specification to `it`, like `it 'returns the defined json_schema' do`